lib/devices/MiioVacuum.js

```diff
@@ -19,10 +19,13 @@ class MiioVacuum {
      * @param {import("../miio/Model")} options.model
      */
     constructor(options) {
-        this.status = new Status();
+        this.robotState = new entities.state.RobotState({
+            map: require("../res/default_map")
```

Comment:
What about removing the "default_map" from the server and add a way for the server to say that he don't have a map yet ? the default map for me should be on the client not the server

Reply:
Then the client would need a way to get the default map (… from the server). So imho there’s little benefit to this.

Reply:
Only if it want to display the default map, i'm developping a alternate client and for example I want to show something else
Without the default map you can know that you don't have a map, with this you don't know because you always have a map

Reply:
https://github.com/Hypfer/Valetudo/blob/18d435a4b26df301cddedc6234e7c2b2bc669576/lib/entities/map/ValetudoMap.js#L22-L23

This should be sufficient